GenericJSONTests/InitializationTests.swift

```diff
@@ -53,4 +53,40 @@ class InitializationTests: XCTestCase {
             "b": true,
         ])
     }
+
+    func testInitializationFromJSONSerilization() throws {
+
+        let jsonData = """
+            {
+                "array": [
+                    1,
+                    2,
+                    3
+                ],
+                "boolean": true,
+                "null": null,
+                "number": 1,
+                "greatest_int": \(Int.max),
+                "greatest_double": \(Double.greatestFiniteMagnitude),
+                "object": {
+                    "a": "b"
+                },
+                "string": "Hello World"
+            }
+            """.data(using: .utf8)!
+
+        let jsonObject = try JSONSerialization.jsonObject(with: jsonData, options: [])
+        let json = try JSON(jsonObject)
+
+        XCTAssertEqual(JSON.array([1, 2, 3]), json["array"]!)
```

Comment:
Could we please flip the order of arguments here? It’s just a matter of taste or convention, but everywhere around we use the `(actual, expected)` order, so it would be nice to keep that here, too.

Reply:
Sure, done.